syntax="proto3";

package fixbug;

message ResultCode
{
    int32 errcode=1;
    bytes errmsg=2;
}

message LoginRequest
{
    bytes name=1;
    bytes pwd=2;
}

message LoginResponse
{
    ResultCode result=1;
    bool success=2;
}

message User
{
    bytes name=1;
    int32 age=2;
}

message userList
{
    repeated User users=1;
}